content/webapp/pages/exhibition-guide.tsx

```diff
@@ -98,9 +115,14 @@ type StopsProps = {
 };
 
 const Stops: FC<StopsProps> = ({ stops, type }) => {
+  const stopWidth = type === 'bsl' ? 'calc(50% - 50px)' : 'calc(33% - 50px)';
   return (
-    <ul className="plain-list no-margin no-padding">
+    <ul
+      className="plain-list no-margin no-padding flex flex--wrap"
+      style={{ gap: '50px' }}
```

Comment:
I really like `gap`. I was wondering if we can find a way to make it work as a responsive space for us (not in this PR though).